bbc_sale/data/action_automated_emails_magento.xml

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<openerp>
+    <data noupdate="1">
```

Comment:
What is exactly the reason you set this to noupdate? It's not like the data in this xml is very volatile or prone to customization, not like the email templates themselves.

Reply:
I see what you mean. My reason to set this to noupdate is as follows: in the XML-file I've set no email templates for the different server actions (we'll set the templates in the user interface afterwards). I thought that as soon as there is a module upgrade of bbc_sale, the email templates are disappeared again in the user interface. But this is in an incorrect thought, I guess?

Reply:
Indeed, we don't need noupdate here. And because the workflow activities are not noupdate in their originating module, the migration script is not necessary either.

Reply:
The reason that the template will not be overridden once you enter it in the UI is that the xml does not contain any value for the template id on the server action.

Reply:
Aha! Now I understand.
I know at least how to make migrations now ;-)